src/main/java/reposense/parser/JsonParser.java

```diff
@@ -20,16 +20,17 @@
 
     /**
      * Converts json file from the given {@code path} into an object and returns it.
-     * @throws FileNotFoundException if {@code path} is invalid.
+     * @throws IOException if {@code path} is invalid.
```

Comment:
is this change necessary?

Reply:
The exception signature has changed after wrapping `JsonReader` with `try-with-resources ` block
i.e. `try (JsonReader jsonReader = new JsonReader(...))`. 

Reason of change is in the reply to @yamidark.